system-tests/README.md

```diff
@@ -19,3 +19,35 @@ It uses Docker containers to create containerised instances of Kafka and other c
 * Wait
 
 Note: these tests take some time to run.
+
+
+### General Architecture
+
+
+The system tests use pytest for the test runner, and use separate fixtures for different configurations of the file-writer. 
+
+Firstly, the system tests attempt to build and tag the latest file-writer image. This can take a lot of time especially if something in conan has changed, as it has to reinstall all of the conan packages.
+
+The Kafka and Zookeeper containers are started with `docker-compose` and persist throughout all of the tests, and when finished will be stopped and removed.
+
+Each fixture starts the file-writer with an `ini` config file (found in `/config-files`), and in some cases use a JSON command at startup so nothing has to be sent over kafka. Some fixtures start the filewriter multiple times and others use the NeXus-streamer image. 
```

Comment:
"so nothing has to be sent over kafka" is misleading. Rather say something like "... in some cases a JSON command, in others the command is sent over Kafka as part of the test.".

Reply:
Say what the NeXus-Streamer is used for: "..use the NeXus-Streamer image as a source of streamed data".